selenium_tests/polls_specific/test_poll_functionality.py

```diff
@@ -69,4 +69,30 @@ def test_result_percentage(self):
         results_page = QuestionnaireResultsPage(self.selenium)
         poll_result = results_page.get_poll_results_text()
         self.assertIn("67 %", poll_result)
-        self.assertIn("33 %", poll_result)
\ No newline at end of file
+        self.assertIn("33 %", poll_result)
+
+    def test_multi_select_option_results_for_checkboxes(self):
```

Comment:
It's checking for the exact bug that was overlooked by everyone, and I believe it's worth keeping it because when we decide to refactor the poll processing, tests like these will save us from different kinds of ripples, and give us confidence in the refactor as well.

Reply:
@sheralim012  This test replicates `test_multi_select_option_results_for_checkboxes` (which covers the bug) and the other test in this class, which is why I'm not convinced that it's useful.

Reply:
@istride The unit test is removed.